google-cloud/docs/toc.json

```diff
@@ -198,7 +198,7 @@
             },
             {
               "title": "Data Types",
-              "type": "google/dlp/v2/datatypes"
+              "type": "google/privacy/dlp/v2/datatypes"
```

Comment:
Why is `google/privacy/dlp` being used as a path? It doesn't exist AFAICT. The only path I see in this library is `google/cloud/dlp`. Shouldn't we replace `google/privacy/dlp` with `google/cloud/dlp`, and `Google::Privacy::Dlp` with `Google::Cloud::Dlp`?

Reply:
If I understood @quartzmo's comment correctly this is to reference the following file `jsondoc/google/privacy/dlp/v2/datatypes.json` in the generated docs.

The generator does produce things in both modules `Google::Privacy` and `Google::Cloud` - is that what you mean?

Reply:
My [comment](https://github.com/GoogleCloudPlatform/google-cloud-ruby/pull/2009#issuecomment-373859288) was that the links need to match the output. I don't personally know which is correct.

Reply:
This value in the Rakefile is what controls the output:

```ruby
title: "Google::Privacy::Dlp::V2::DataTypes",
```
But I don't know if `Google::Privacy::Dlp::V2::DataTypes` is really the correct title. I don't know where it came from. Was it generated?

Reply:
The history of "Data Types" is that someone wanted a "page" with a table listing all the "minor" types such as protobufs. Then that person or someone else requested that minor types from our manual API should appear there as well. The Angular app was never designed to generate a page like this, so I added the generation to our jsondoc tool, controlled by the config in the Rakefile. It's pretty simple, and flexible.

Reply:
I believe `Google::Privacy::Dlp::V2::DataTypes` is generated. All the generated protobuf classes are under `Google::Privacy` instead of `Google::Cloud` because of how it was setup. So AFAIK this is consistent.

Reply:
OK, SGTM.